### PR TITLE
docs: mention Fluentd is deprecated in values file

### PR DESCRIPTION
**Pull request by astencel-sumo**

### CODE REVIEW

File: deploy/helm/sumologic/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
...
+| `sumologic.events.provider`                                                      | Defines which provider is used for Kubernetes events collection - `otelcol` or `fluentd`. Otelcol is the default and is recommended. Fluentd is deprecated.                                                                                                                                                                                                          | `otelcol`                                                                                                                                                                            |
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/README.md:30]: I suspect this configuration is case sensitive

```suggestion
| `sumologic.events.provider`                                                      | Defines which provider is used for Kubernetes events collection - `otelcol` or `fluentd`. `otelcol` is the default and is recommended. `fluentd` is deprecated.                                                                                                                                                                                                      | `otelcol`                                                                                                                                                                            |
```

File: deploy/helm/sumologic/README.md [hunk 2]
```diff
@@ -82,9 +82,9 @@ The following table lists the configurable parameters of the Sumo Logic chart an
...
+| `sumologic.logs.metadata.provider`                                               | Defines logs metadata enrichment provider - `otelcol` or `fluentd`. Otelcol is the default and is recommended. Fluentd is deprecated.                                                                                                                                                                                                                                | `otelcol`                                                                                                                                                                            |
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/README.md:85]:
```suggestion
| `sumologic.logs.metadata.provider`                                               | Defines logs metadata enrichment provider - `otelcol` or `fluentd`. `otelcol` is the default and is recommended. `fluentd` is deprecated.                                                                                                                                                                                                                            | `otelcol`                                                                                                                                                                            |
```

File: deploy/helm/sumologic/README.md [hunk 3]
```diff
@@ -82,9 +82,9 @@ The following table lists the configurable parameters of the Sumo Logic chart an
...
+| `sumologic.metrics.metadata.provider`                                            | Defines metrics metadata enrichment provider - `otelcol` or `fluentd`. Otelcol is the default and is recommended. Fluentd is deprecated.                                                                                                                                                                                                                             | `otelcol`                                                                                                                                                                            |
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/README.md:87]:
```suggestion
| `sumologic.metrics.metadata.provider`                                            | Defines metrics metadata enrichment provider - `otelcol` or `fluentd`. `otelcol` is the default and is recommended. `fluentd` is deprecated.                                                                                                                                                                                                                         | `otelcol`                                                                                                                                                                            |
```

File: deploy/helm/sumologic/values.yaml [hunk 4]
```diff
@@ -235,7 +236,7 @@ sumologic:
   logs:
     enabled: true
     metadata:
-      ## Set provider service (either fluentd or otelcol).
+      ## Defines logs metadata enrichment provider - `otelcol` or `fluentd`. Otelcol is the default and is recommended. Fluentd is deprecated.
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/values.yaml:239]:
```suggestion
      ## Defines logs metadata enrichment provider - `otelcol` or `fluentd`. `otelcol` is the default and is recommended. `fluentd` is deprecated.
```

File: deploy/helm/sumologic/README.md (hunk 1 above)
Comment (astencel-sumo) [reply deploy/helm/sumologic/README.md]: Changed

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**